src/Features/Core/Portable/ChangeSignature/CannotChangeSignatureReason.cs

```diff
@@ -9,6 +9,7 @@ internal enum CannotChangeSignatureReason
         None,
         DefinedInMetadata,
         IncorrectKind,
-        InsufficientParameters
+        DeclarationLanguageServiceNotFound,
```

Comment:
=-o

When does this happen??

Reply:
Maybe if the declaration is in F#? @ivanbasov Do you remember?

Reply:
yeah, can this be tossed?

Reply:
Yep. I already commented out the uses of it in the next PR (before deleting), but I'll remove these entries as well.